ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main__ocp4.14-amd64.yaml

```diff
@@ -0,0 +1,81 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  operator-sdk:
+    name: "4.14"
+    namespace: origin
+    tag: operator-sdk
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+canonical_go_repository: github.com/grafana/tempo-operator
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: tempo-operator
+- context_dir: .
+  dockerfile_path: tests/Dockerfile
+  to: tempo-tests-runner
+operator:
+  bundles:
+  - as: tempo-bundle
+    context_dir: ./bundle/openshift
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: ghcr.io/grafana/tempo-operator/tempo-operator:v0.2.0
+    with: pipeline:tempo-operator
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: tempo-e2e-tests
+  cron: 0 6 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
```

Comment:
Can we use a different cluster_profile for this upstream pre-submit testing. Our team's aws-cspi-qe cluster_profile is only budgeted to run periodic testing once a week.